### ISSUE TITLE
Feature request: make F8 always save

### ISSUE THREAD

**Issue by grippon** [geany/geany#1050]
I like the F8 shortcut which allow to save and check syntax with a local tool.

But I would like F8 to save the file even if nothing is defined to check the syntax.

Is it possible?

**Comment by elextr** [comment]
It is possible for the build commands to be changed to do the save before checking the command, but it doesn't seem particularly necessary, there are plenty of other ways of saving a file.  And you can always re-bind f8 to save.

A well written pull request that made it an option (since I can also see people not wanting it to happen) might be considered.

**Comment by codebrainz** [comment]
Or just set the command to `echo | /dev/null` or something with no effect.

**Comment by grippon** [comment]
Yes it works with the command `true` but I have to configure that for each filetype?
Or there is a shortcut?

Also I don't want to think "wait this file has no syntax checker so I should press Ctrl+S"
If I rebind F8 to save I won't have syntax check anymore.

I see F8 as "save file and applies syntax check if there is"

**Comment by elextr** [comment]
See [this](https://wiki.geany.org/howtos/configurebuildmenu) for details on configuring build menu commands.  Anything you set will override the default filetype settings (which is as it should be), so yes you will have to set it for each filetype you want to behave that way..
